server/models/migrations/next/migrate.sql

```diff
@@ -6,4 +6,17 @@ INSERT INTO unit VALUES
 (268, '[Stock] Consumption graph','TREE.STOCK_CONSUMPTION_GRAPH_REPORT','Stock Consumption graph report', 144,'/modules/reports/generated/stock_consumption_graph_report','/reports/stock_consumption_graph_report');
   
 INSERT INTO `report` (`report_key`, `title_key`) VALUES
-('stock_consumption_graph_report', 'REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE');
\ No newline at end of file
+('stock_consumption_graph_report', 'REPORT.STOCK_CONSUMPTION_GRAPH_REPORT.TITLE');
+
+
+--@lomamech 2020-04-25
+ALTER TABLE `stock_movement` ADD COLUMN `period_id` MEDIUMINT(8) UNSIGNED DEFAULT NULL;
```

Comment:
I don't think this should ever be NULL.  It doesn't make sense to record a stock movement for a day that we don't have a period defined.